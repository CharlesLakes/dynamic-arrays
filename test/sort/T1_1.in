10
94 60 48 42 37 79 61 46 45 38