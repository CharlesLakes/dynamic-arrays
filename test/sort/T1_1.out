37 38 42 45 46 48 60 61 79 94